ts/masters-es5/polyfills.ts

```diff
@@ -57,7 +57,10 @@ if (!Object.values) {
 }
 
 (function () {
-    if (typeof window.CustomEvent === "function") return false;
+    if (
+        !document || !window ||
```

Comment:
I think it can still throw an error in a non-browser environment. Have you tried something like:
```suggestion
        !typeof document !== 'undefined' ||
        !typeof window !== 'undefined' ||
```

Reply:
I optimized the statements.